src/Compilers/CSharp/Test/Syntax/Parsing/ParsingTests.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-//#define PARSING_TESTS_DUMP
+#define PARSING_TESTS_DUMP
```

Comment:
Please revert

Reply:
Will do.  it's temporary as i add tests :D